sfepy/terms/terms_surface.py

```diff
@@ -18,9 +19,10 @@ class LinearTractionTerm(Term):
     :math:`\ul{f}` for a traction vector, and itself for a stress tensor.
 
     The material parameter can have one of the following shapes: 1 or (1, 1),
-    (D, 1), (S, 1). The symmetric tensor storage is used in the last case: in
-    3D S = 6 and the indices ordered as :math:`[11, 22, 33, 12, 13, 23]`, in 2D
-    S = 3 and the indices ordered as :math:`[11, 22, 12]`.
+    (D, 1), (S, 1), or (D, D) in the `eval` mode. The symmetric tensor storage
+    is used in the last case: in 3D S = 6 and the indices ordered as
```

Comment:
Make (S, 1) last so that it is the last case.

Reply:
`(D, D)` is possible in  the `eval` mode only, so I modified the docstring in this way.

Reply:
OK, so what about
```
The symmetric tensor storage (S, 1) is as follows: in 3D S = 6 and the indices ordered as
```

Reply:
Yes, I see. I did not noticed "in the last case".